docs/src/architecture/08_concepts/catalyst_voting/transaction.md

```diff
@@ -115,7 +114,56 @@ Expected witness (includes signature)
 
 <!-- markdownlint-disable code-block-style -->
 ```hex
-    0200000000e6c8aa48925e37fdab75db13aca7c4f39068e12eeb3af8fd1f342005cae5ab9a1ef5344fab2374e9436a67f57041899693d333610dfe785d329988736797950d
+0200000000e6c8aa48925e37fdab75db13aca7c4f39068e12eeb3af8fd1f342005cae5ab9a1ef5344fab2374e9436a67f57041899693d333610dfe785d329988736797950d
+```
+<!-- markdownlint-enable code-block-style -->
+
+### v2
+
+<!-- markdownlint-disable max-one-sentence-per-line code-block-style -->
+??? note "V2 vote transaction definition: `tx_v2.cddl`"
+
+    ```CDDL
+    {{ include_file('src/architecture/08_concepts/catalyst_voting/tx_v2.cddl', indent=4) }}
+    ```
+<!-- markdownlint-enable max-one-sentence-per-line code-block-style -->
+
+`event` - a set of different identifiers which is uniquely define a particular voting event.
+
+Vote:
+
+* `choices` - a collection of voter choices for the proposal.
+* `proof` - a voter proof, could be `null`.
+* `prop_id` - a proposal id for which `choice` is made, could be `null`.
+  For that case where for the voting event defined only **one** proposal,
```

Comment:
for the voting `event`

So the reader knows that this refers to the `event` field